### PR TITLE
Add rvagg and jennijuju as members to "github-mgmt stewards"

### PR DESCRIPTION
**Pull request by BigLep**
### Summary
Add rvagg as a member to "githut-mgmt stewards"

Also started documented why the current set of members are there.

### Why do you need this?
This is being done now because there have been various permissions/access issues of late as part of cleaning up repos that FilOz owns/maintainers (e.g., improving CI, updating dependencies, adding dependabot).  This empowers FilOz to start attemping some better practice of doing these kind of permissions changes under PR and in a more transparent way.

### What else do we need to know?
This set of members will also get reviewed and improved as part of https://github.com/filecoin-project/github-mgmt/issues/47

### Reviewer's Checklist

- [ ] It is clear where the request is coming from (if unsure, ask)
- [ ] All the automated checks passed
- [ ] The YAML changes reflect the summary of the request
- [ ] The Terraform plan posted as a comment reflects the summary of the request

### CODE REVIEW

**Comment by BigLep** [comment]
FYI @rvagg that I am adding you to github-mgmt stewards.

**Comment by BigLep** [comment]
I'm going to go ahead and merge this because:
1. @galargh did have a look and didn't request changes
2. we can revert/edit this if there are concerns or followup feedback

File: github/filecoin-project.yml [hunk 1]
```diff
@@ -5206,10 +5206,32 @@ teams:
...
       #  - be familiar with GitHub Management
       #  - be ready to triage/review org configuration change request in github-mgmt
+      # The individuals below are listed as "maintainers" rather than "members" because they are filecoin-project owners/admins.
+      # GitHub will auto-bump their team privileges anyway if we don't manually.
       maintainer:
+        # Why @galargh?
+        # 1. He has deep knowledge of the tool and its use as the creator.
+        # 2. He is co-founder of IPDX, which has a contract in 2024 to support developer productivity in the filecoin-project ecosystem.
         - galargh
+        # Why @jennijuju?
```
Comment (jennijuju) [inline github/filecoin-project.yml:5216]: Hrm I guess I’m fine with this but also hoping there are more people that is stewarding this so im not blocking any urgent requests. TLDR maybe onboard and  get two or more people from the project owner group to this workflow

**Review by jennijuju — APPROVED** [review]

File: github/filecoin-project.yml (hunk 1 above)
Comment (BigLep) [reply github/filecoin-project.yml:5216]: Yes, that is the plan.  We'll get some others when holistically attacking org ownership in https://github.com/filecoin-project/github-mgmt/issues/47 .  (This group needs to be looked at in light of who is also an org owner/admin as this group can self-service elevant themselves and the org admin/owners can self-service make changes here as well.).

**Pull request merged**